src/commands/revenue.js

```diff
@@ -35,6 +37,13 @@ function definePeriods(args, options) {
     periods.push([start, end]);
   }
 
+  function createYearPeriod(year) {
+    const start = moment(year, 'YYYY').utc();
```

Comment:
I was expecting to obtain from this the zero hours in UTC, but what I get is the zero hour in my time zone plus the difference (+2 hours). Is this correct or it would skip reviews that were done, fro example, at 2017-01-01 at 01h in UTC?